### PR TITLE
XS-4195

### PR DESCRIPTION
**Pull request by allenbross-wf**
### Changes:

- Implemented new rule
- Incremented version number

### Pull request process

- [x] Full description of changes provided above.
- [x] Reviewed by another person. (+1)
- [ ] If this is a code change, reviewed by a second person. (+1)
- [ ] QA'd to the specifications listed [here](https://github.com/DataQualityCommittee/dqc_us_rules#quality-assurance-qa-of-a-pull-request). Documentation only changes do not require the formal code quality assurance process. (+10)

### CODE REVIEW

**Comment by jasonleinbach-wf** [comment]
+1

File: dqc_us_rules/dqc_us_0008.py [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+        del calc_loading_instance
+        year += 1
+
+
+def _reorder_dictionary(calc_children, calc_children_ordered):
+    """
+    :param calc_children:
+    :type calc_children:
+    :param calc_children_ordered:
+    :type calc_children_ordered:
```
Comment (joyreistad-wf) [inline dqc_us_rules/dqc_us_0008.py]: missing comments

**Comment by joyreistad-wf** [comment]
+1

**Comment by jasonleinbach-wf** [comment]
+1

**Comment by joyreistad-wf** [comment]
+1

**Comment by chrislococo-wf** [comment]
+1

**Pull request closed without merging**